src/server/session/stateless-session-store.test.ts

```diff
@@ -582,6 +645,71 @@ describe("Stateless Session Store", async () => {
         expect(cookie?.secure).toEqual(true);
       });
     });
+
+    it("should set new cookie and delete legacy cookie if both exist (chunked)", async () => {
```

Comment:
What does both exist refer to here? I was assuming it meant both new and legacy cookie would exist, but unless I am mistaken, the test only sets the legacy cookie (which I think is exected)?